jib-core/src/test/java/com/google/cloud/tools/jib/cache/CacheStorageReaderTest.java

```diff
@@ -638,4 +638,52 @@ public void testVerifyImageMetadata_validOciImageIndex() throws CacheCorruptedEx
     CacheStorageReader.verifyImageMetadata(metadata, Paths.get("/cache/dir"));
     // should pass without CacheCorruptedException
   }
+
+  @Test
+  public void testAllLayersCached_v21SingleManifest()
+      throws IOException, CacheCorruptedException, DigestException, URISyntaxException {
+
+    setupCachedMetadataV21(cacheDirectory);
```

Comment:
Minor suggestion: If possible, let's format the test into three blocks: arrange, act and assert, with a space between each of these blocks. The `arrange` block will take care of all setup, the `act` block will call the method we're testing and the `assert` block will do all the verification.

Reply:
Thanks for the suggestion, this makes a lot of sense! Tried to follow this idea though with the `act` and `assert` block more or less combined, since for a few of the tests had asserts both before and after the action.

Reply:
How about we store these calls in variables? For example:

```
cacheAfterFirstLayerDirectory = Files.createDirectories(cacheStorageFiles.getLayerDirectory(firstLayerDigest));
areAllLayersCachedAfterFirstLayer = cacheStorageReader.areAllLayersCached(manifest)
cacheAfterSecondLayerDirectory = ...
areAllLayersCachedAfterSecondLayer= ..

//Assert block
```

This is just an example so you can pick a name you think works better.